test/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ option(JSON_Sanitizer "Build test suite with Clang sanitizer" OFF)
 option(JSON_Valgrind "Execute test suite with Valgrind" OFF)
 option(JSON_NoExceptions "Build test suite without exceptions" OFF)
 option(JSON_Coverage "Build test suite with coverage information" OFF)
+option(JSON_ForceCpp14 "Use C++14 standard" OFF)
```

Comment:
This flag is not used in Travis or AppVeyor, right?

Reply:
True, I will remove it.

Reply:
Ok.